app/controllers/api/v1/events_by_month_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class EventsByMonthController < Api::V1::ApiController
+      def index
+        date = params[:date] ? Time.zone.parse(params[:date]) : Time.zone.now
+        @events = Event.on_month(date, current_api_v1_user)
+        @calendar = events_to_calendar_view(@events)
+        raise ActiveRecord::RecordNotFound, 'not found' unless @events
```

Comment:
Why don't we just return a month with no events?

Reply:
Ok